kedro/extras/datasets/matplotlib/matplotlib_writer.py

```diff
@@ -27,51 +27,81 @@ class MatplotlibWriter(
     """``MatplotlibWriter`` saves one or more Matplotlib objects as
     image files to an underlying filesystem (e.g. local, S3, GCS).
 
-    Example:
+    Example adding a catalog entry with the `YAML API
+    <https://kedro.readthedocs.io/en/stable/data/\
+        data_catalog.html#use-the-data-catalog-with-the-yaml-api>`_:
+
+    .. code-block:: yaml
+
+        >>> output_plot:
+        >>>   type: matplotlib.MatplotlibWriter
+        >>>   filepath: data/08_reporting/output_plot.png
```

Comment:
It might be useful to show how you can add `save_args` in yaml, so new users realise that option exists.

Reply:
Done!